sorc/ncep_post.fd/CMakeLists.txt

```diff
@@ -171,7 +171,7 @@ if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
   set(CMAKE_Fortran_FLAGS
       "-g -traceback -fp-model source -free -convert big_endian")
   set(CMAKE_Fortran_FLAGS_RELEASE "-O3")
-  set(CMAKE_Fortran_FLAGS_DEBUG "-O0 -check bounds -check pointers -check shape -check stack -check uninit")
+  set(CMAKE_Fortran_FLAGS_DEBUG "${CMAKE_Fortran_FLAGS_DEBUG} -O0")
```

Comment:
```suggestion
  set(CMAKE_Fortran_FLAGS_DEBUG "-O0 -check all -ftrapuv")
```

Reply:
I do believe that a higher level project should be able to change all their sub-component level flags if needed through CMAKE_Fortran_FLAGS_<build>, and I'm not arguing that this (your) way is incorrect/wrong, but maybe in the future using `isSubProject` and allowing that to happen would be worthwile.

Ignoring what a high level project might need to build with doesn't seem like the best way.